scripts/game/game_storage.js

```diff
@@ -205,6 +205,15 @@ class GameStorage {
   }
 
   storeAPI(data){
+
+    if (data.Level === null && data.Event === 17) {
```

Comment:
This duplicates isGameOver check on line 151. If the issue is "errorUnknown" message then I'd go with updating getErrorBlock() to return empty object for all these error id's (16-22).

Reply:
probably instead we can change `isError` method.

according to API, it should look like this:

```javascript
  isError(){
    return !([0, 16,18,19,20,21,22].includes(this.last.Event));
  }
```